llvm/lib/Transforms/IPO/GlobalOpt.cpp

```diff
@@ -2043,9 +2043,13 @@ static bool callInstIsMemcpy(CallInst *CI) {
 }
 
 static bool destArrayCanBeWidened(CallInst *CI) {
+  auto *GV = dyn_cast<GlobalVariable>(CI->getArgOperand(0));
```

Comment:
getArgOperand(1)

Reply:
Done

Reply:
Removed as not necessary after reviewing